File: tests/integration/features/conditionals/conditionals.java
// Conditional shape: flat <if>/<else_if>/<else>. Ternary keeps
// <then>/<else> via surgical wrap.

class Conditionals {
    String classify(int n) {
        if (n < 0) {
            return "neg";
        } else if (n == 0) {
            return "zero";
        } else if (n < 10) {
            return "small";
        } else {
            return "big";
        }
    }

    String label(int n) {
        return n > 0 ? "positive" : "non-positive";
    }
}

File: tests/integration/features/conditionals/conditionals.py
# Conditional shape: `elif` renders as <else_if> (underscore naming per
# Principle #1), collapsed as a flat sibling of <if>. Ternary expression
# keeps <then>/<else> wrappers via surgical field wrap.

def classify(n):
    if n < 0:
        return "neg"
    elif n == 0:
        return "zero"
    elif n < 10:
        return "small"
    else:
        return "big"


def label(n):
    return "positive" if n > 0 else "non-positive"
